### ISSUE TITLE
"user" field hidden, but used for login

### ISSUE THREAD

**Issue by clemisch** [mssun/passforios#611]
The "user: " field is hidden in the UI. Confusingly, it is being used when autofilling in Safari, instead of the "login" field. Is this intended behavior? 

I would have expected:

1. the "user" field being shown in the UI,
2. the "login" field being used for autofill, even if "user" field is present.

![5393966F-CDC1-4066-8896-B8CB09416593](https://user-images.githubusercontent.com/5190547/227481326-c6661b28-78c0-4d58-a843-f116567c3457.jpeg)
![680D605A-43EF-454B-94F9-1EE15E963CB2](https://user-images.githubusercontent.com/5190547/227481335-7c6e6b7b-6346-47c3-9c4c-5c6f06a526d9.jpeg)

**Comment by brother** [comment]
This sounds like two separate issues to me. 
the hidden user field is one and that it is used instead of login (might be debatable what field to use there… add a setting perhaps?).

the hidden user is actually super annoying as it’s not something we can toggle.

**Comment by clemisch** [comment]
Yes, I agree with you. It's still relevant to discuss them together since the user field being hidden but used can be very confusing.

**Comment by clemisch** [comment]
I think the corresponing commit is 3ab334dd00401d087d9f6618c9bfce8570f54ea7. It does not seem to me that the prevalence of `user` over `login` is intended.

**Comment by chrisgrieser** [comment]
Somewhat related, using "username" as field results in "user" as well as "username" being displayed. 

The only key that supports autofill and is displayed correctly seems to be "login".

**Comment by c0dev0id** [comment]
I stubled over this too. Some accounts have a username, but the email is used for login.

I find it weird that "user" takes precedence over "login".
